adoptopenjdk-api-v3-utils/src/main/kotlin/net/adoptopenjdk/api/v3/ai/AppInsightsTelemetery.kt

```diff
@@ -1,19 +1,23 @@
-package net.adoptopenjdk.api.v3.metrics
+package net.adoptopenjdk.api.v3.ai
 
 import com.microsoft.applicationinsights.TelemetryClient
 import java.io.File
 import java.io.FileOutputStream
 import java.nio.file.Files
 import java.nio.file.Path
 
-internal object AppInsightsTelemetery {
+object AppInsightsTelemetery {
 
-    internal val telemetryClient: TelemetryClient
+    val telemetryClient: TelemetryClient
 
     init {
         telemetryClient = loadTelemetryClient()
     }
 
+    fun start() {
```

Comment:
What's the value of the empty function? I can see from its usage that it's essentially a way to make a call to this `AppInsightsTelemetery ` object and therefore kick-off its initialisation so, if it's required, maybe adding some logging to `start` will mean it's adding some value by providing feedback on the initialisation process.

Reply:
we're probably also working around the dependency injection/wiring constraints we currently have as i might be nicer for this to be class that we initialise and inject - but that's one for another day

Reply:
yeah, some function where we can make sure that the object has been initialised is the idea for this function. I am not sure if kotlin has a proper way to do this, but we are effectively create an eager singleton, which as you say DI would allow us to address properly